ci-operator/config/openshift-logging/interop-ocp-ci/openshift-logging-interop-ocp-ci-main__hypershift-4.14-lp-rosa-hypershift.yaml

```diff
@@ -0,0 +1,118 @@
+base_images:
```

Comment:
This DOES NOT MEET the conventions of using the release repo.  I will re-iterate as expressed in the config docs this file must be formatted as such:

```
config/
  |- organization_name
       |- repo_name
          |- organization_name-repo_name-branch_name.yaml
```
Currently this says you have some repo in the openshift-logging org which [clearly](https://github.com/openshift-logging) there are no repos established

Reply:
yes. there is no real openshift-logging/interop-ocp-ci repo.   The test code  is in https://github.com/openshift/openshift-tests-private.  as openshift-tests-private is private repo which can not shared to interop team.  that is why we want to create a public folder  here.   For there isn't real repo, i think we can ignore the definition about branch.

Reply:
@jcantrill , we did this because we can't use openshift-tests-private since we don't have access to private deck, and as per the communication rececived we also can't use cluster-logging repo
So we were suggested to use a dummy repo openshift-logging/interop-ocp-ci

Reply:
here is the onboarding doc for guidlines that we follow - https://cspi.gitbook.io/ocp-ci-onboarding